### PR TITLE
refactor(getting-started): use new invite members component

### PR DESCRIPTION
**Pull request by H4ad**
## Pull Request

---

This uses the component introduced at #11872 to simplify the code around invite members.

Before:

https://github.com/user-attachments/assets/51699fb7-3813-45bd-a8d3-64980b79cb5a

After:

https://github.com/user-attachments/assets/5c852bf2-4313-4890-8766-52ae158ab44a

Part of https://github.com/SigNoz/platform-pod/issues/2279

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [ ] 🐛 Bug fix
- [x] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

- Tests added/updated: Yes
- Manual verification: No
- Edge cases covered: -

---

- Blast radius: Getting Started - Invite Team Members
- Potential regressions: -
- Rollback plan: Revert this commit.

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Feature |
| Description | We updated the Getting Started flow around Invite Team Members to have a common experience around invite member across the product. |

---

### 📋 Checklist
- [x] Tests added or explicitly not required
- [x] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

### CODE REVIEW

File: frontend/src/container/OnboardingV2Container/AddDataSource/AddDataSource.tsx [hunk 1]
```diff
@@ -1094,12 +1099,55 @@ function OnboardingAddDataSource(): JSX.Element {
...
+							}}
+							onAllFailed={(): void => {
+								void logEvent(
+									`${ONBOARDING_V3_ANALYTICS_EVENTS_MAP?.BASE}: ${ONBOARDING_V3_ANALYTICS_EVENTS_MAP?.INVITE_TEAM_MEMBER_FAILED}`,
+									{},
+								);
+							}}
+							renderFooter={({ submit, canSubmit, isSubmitting }): JSX.Element => (
+								<div className="invite-team-member-modal-footer">
+									<Button
```
Comment (AshwinBhatkal) [inline frontend/src/container/OnboardingV2Container/AddDataSource/AddDataSource.tsx]: use signoz buttons for this? Named alias

**Review by AshwinBhatkal — APPROVED** [review]

**Review by makeavish — APPROVED** [review]
